link-check/steps.sh

```diff
@@ -20,6 +22,6 @@ echo "Checking links"
   xargs -0 /liche ${INPUT_DOC_ROOT:+-d "${INPUT_DOC_ROOT}"} \
          ${INPUT_TIMEOUT:+-t "${INPUT_TIMEOUT}"} \
          ${INPUT_NUM_REQUESTS:+-c "${INPUT_NUM_REQUESTS}"} \
-         ${INPUT_EXCLUDE_URLS:+-x "${INPUT_EXCLUDE_URLS}"}
-         ${INPUT_VERBOSE:+-v} \
-) && echo "No broken links!"
+         ${INPUT_EXCLUDE_URLS:+-x "${INPUT_EXCLUDE_URLS}"} \
+         ${INPUT_VERBOSE:+-v} ) \
```

Comment:
I can't see an opening `(` anywhere, is that needed?

Reply:
It's in line 19 (the diff doesn't show it)